src/Sciences.API/Repositories/ITopicRepository.cs

```diff
@@ -7,8 +7,8 @@ namespace Sciences.API.Repositories;
 public interface ITopicRepository
 {
     Task<List<Topic>> GetTopics();
-    Task AddTopic(CreateTopicModel model);
-    Task UpdateTopic(Guid topicId, UpdateTopicModel model);
-    Task DeleteTopic(Guid topicId);
-    Task<Topic> GetTopicById(Guid topicId);
+    Task <TopicModel> AddTopic(Guid scienceId,CreateTopicModel model);
+    Task UpdateTopic(Guid scienceId, Guid topicId, UpdateTopicModel model);
+    Task DeleteTopic(Guid scienceId,Guid topicId);
+    Task<Topic> GetTopicById(Guid scienceId,Guid topicId);
```

Comment:
hammasida DTO model kirib chiqishi kerak

Reply:
> hammasida DTO model kirib chiqishi kerak

to'g'rilandi

Reply:
Ustoz repository data base bn ishlaydiku unga faqat entity kirib entity chiqadi managerda model kirib model chiqadigan qilishim kerak edi shunda umuman notogri qilibman